instrumentation/rocketmq-client-4.8/javaagent/rocketmq-client-4.8-javaagent.gradle

```diff
@@ -0,0 +1,16 @@
+apply from: "$rootDir/gradle/instrumentation.gradle"
+
+muzzle {
+  pass {
+    group = "org.apache.rocketmq"
+    module = 'rocketmq-client'
+    versions = "[4.8.0,)"
```

Comment:
Can you add `assertInverse = true` to make sure that all other versions are not instrumented?